akka-docs/src/main/paradox/typed/cluster-sharding.md

```diff
@@ -64,24 +64,36 @@ Java
 When using sharding entities can be moved to different nodes in the cluster. Persistence can be used to recover the state of
 an actor after it has moved.
 
-Taking the larger example from the @ref:[persistence documentation](persistence.md#larger-example) and making it into
-a sharded entity is the same as for a non persistent behavior. The behavior:
+Akka Persistence is based on the single-writer principle the persistent actors are typically used together
```

Comment:
I found this sentence hard to parse. Maybe:

Akka Persistence is based on the single-writer principle, for a particular `persitenceId` only one persistent actor instance should be active. If multiple instances were to persist events at the same time, the events would be interleaved and might not be interpreted correctly on replay. Cluster sharding is typically used together with persistence to ensure that there is only one active entity for each `persistenceId`.

Reply:
👍 thanks